openedx/core/djangoapps/session_inactivity_timeout/middleware.py

```diff
@@ -14,12 +14,16 @@
 
 
 from datetime import datetime, timedelta
+import logging
 
 from django.conf import settings
 from django.contrib import auth
 from django.utils.deprecation import MiddlewareMixin
 
 LAST_TOUCH_KEYNAME = 'SessionInactivityTimeout:last_touch'
```

Comment:
Would it make sense to change the key, since the format of the values will be different?
```suggestion
LAST_TOUCH_KEYNAME = 'SessionInactivityTimeout:last_touch_str'
```

Reply:
updated